ldes-server-port-ingest/pom.xml

```diff
@@ -55,6 +55,16 @@
             <groupId>org.assertj</groupId>
             <artifactId>assertj-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-classic</artifactId>
+            <version>${logback-classic.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>ch.qos.logback</groupId>
+            <artifactId>logback-core</artifactId>
+            <version>${logback-core.version}</version>
+        </dependency>
```

Comment:
Are these deps required? Previously, we already had logging and didn't need these deps?

Reply:
testing if message is logged

Reply:
Sorry to interrupt, but I was wondering why you want to test the logging? In my opinion, I think it is sufficient to check whether the eventPublisher has been called and so the dependencies can be removed again :)

Reply:
ok overkill , it has been removed